pkg/certificate/providers/tresor/types.go

```diff
@@ -42,6 +40,9 @@ type CertManager struct {
 	cacheLock sync.Mutex
 
 	certificatesOrganization string
+
+	// Dynamic config retrieval
```

Comment:
I don't see this comment adding much value given that the Configurator interface is already self explanatory.

Reply:
Will remove in a follow-up.